### PR TITLE
rgw: multisite

### PR DESCRIPTION
**Pull request by theanalyst**
#94 rebased after ganesha merge

### CODE REVIEW

File: srv/salt/ceph/rgw/multisite-default.sls [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Maybe this should be in init, as this is _always_ required
+
+install rgw:
+  pkg.installed:
+    - name: ceph-radosgw
+
+{% is_master = salt['pillar.get')('rgw_is_master')
```
Comment (BlaineEXE) [inline srv/salt/ceph/rgw/multisite-default.sls]: It looks like a `%}` is missing from the end of this line.

Comment (theanalyst) [inline srv/salt/ceph/rgw/multisite-default.sls]: thanks! fixing, didn't test after rebase yet

**Comment by swiftgist** [comment]
@theanalyst Were you going to go with the hierarchical structure with this?

**Comment by swiftgist** [comment]
Continuing discussion on #229

**Pull request closed without merging**